### DISCUSSION TITLE
支付宝使用category作为账单相关的配置

### DISCUSSION THREAD

**Discussion by ppodxiang** [deb-sig/double-entry-generator#36] [Ideas]
现在支付宝很准确的对交易（消费）进行了分类，比如服装、美食、汽车等，完全可以替代peer或者item来作为expense分类，考虑更新支持么？

**Comment by Triple-Z** [comment]
可以在支付宝上提供 category 过滤，不过这个属性只存在于 alipay 了（目前 wechat 也无法支持），需要在文档里标注这一点。

Welcome to PR~

**Reply by gaocegege** [reply]
+1 感觉是一个非常不错的需求（我也想要

**Reply by gaocegege** [reply]
https://github.com/deb-sig/double-entry-generator/issues/37

**Comment by Triple-Z** [comment]
https://github.com/deb-sig/double-entry-generator/releases/tag/v1.2.0

已经支持。
